GPU/GPUTracking/Base/GPUParam.inc

```diff
@@ -41,11 +41,28 @@ GPUdi() void MEM_LG(GPUParam)::Global2Slice(int iSlice, float X, float Y, float
   *z = Z;
 }
 
+#ifdef GPUCA_TPC_GEOMETRY_O2
+
 MEM_CLASS_PRE()
-GPUdi() float MEM_LG(GPUParam)::GetClusterErrorSeeding(int yz, int type, float z, float angle2) const
+GPUdi() void MEM_LG(GPUParam)::GetClusterErrorsSeeding2(int iRow, float z, float sinPhi, float DzDs, float& ErrY2, float& ErrZ2) const
+{
+  GetClusterErrors2(iRow, z, sinPhi, DzDs, ErrY2, ErrZ2);
+}
+
+MEM_CLASS_PRE()
+GPUdi() float MEM_LG(GPUParam)::GetClusterError2(int yz, int type, float z, float angle2) const
 {
-  //* recalculate the cluster ErrorsSeeding wih respect to the track slope
+  MakeType(const float*) c = ParamErrors[yz][type];
+  float v = c[0] + c[1] * angle2 + c[2] * z + c[3] * 0;
+  v = CAMath::Abs(v);
+  return v;
+}
+
+#else // GPUCA_TPC_GEOMETRY_O2
 
+MEM_CLASS_PRE()
+GPUdi() float MEM_LG(GPUParam)::GetClusterErrorSeeding(int yz, int type, float z, float angle2) const
+{
   MakeType(const float*) c = ParamErrorsSeeding0[yz][type];
   float v = c[0] + c[1] * z + c[2] * angle2;
```

Comment:
Is it correct that both error^2 and errorSeeding linearly depend on Z? I would expect the error (not squared) to depend on sqrt(z)

Reply:
@shahor02 : Well, the SeedingError is from Run 2 data, where we had 2 parameterizations. That is what we have been using all the time, so I assume it is correct.

For the Run3 data, since Marian provided only 1 parameterization, I set them to both use the same. We still need to check if this doesn't decrease the efficiency or so.

And in Marians current formula: https://gitlab.cern.ch/alice-tpc-offline/alice-tpc-notes/-/blob/fc3fda4a1954be2e7ca7416c90a2c5543b94b723/JIRA/ATO-630/ATO-630_ErrParam.C#L13
sigma2 depends linearly on Z.

Reply:
One of them should be wrong. Since the sigma increases due to the diffusion, I assume the sigma^2 linear dependence on Z is correct so the SeedingError ~ z should be wrong...

Reply:
I also think so. This seeding error was actually wrong parameterization used by Sergey at some point, and when he fixed it with the correct one we say that the track seeding efficiency went down, so we split the error parameterization in the one for seeding and for the track fit. So this is probably not a realistic cluster error, but some estimate that works well in seeding / track following.
Anyway, it is only for Run 2 data. For Run 3 we need to check what we'll do.

Reply:
Ok, indeed, now I recall this story, we should probably try with the same error as for fitting plus some margin.